pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTValue.java

```diff
@@ -31,7 +31,7 @@
  * @author Andrew Bayer
  * @author Kohsuke Kawaguchi
  */
-public abstract class ModelASTValue extends ModelASTElement implements ModelASTMethodArg {
+public abstract class ModelASTValue extends ModelASTElement implements ModelASTMethodArg, ModelASTEnvironmentValue {
```

Comment:
Sorta agree with @rsandell here, if every `ModelASTValue` _is_ a `ModelASTEnvironmentValue` what's the point in differentiating them?